buildout_1.rst

```diff
@@ -342,7 +330,13 @@ Be McGuyver
 
         $ ./bin/buildout -v
 
-    If strange egg versions are requested, check the dependencies declaration of your eggs and your version pinnings.
+    If strange egg versions are requested, check the dependencies declaration of your eggs and your version pinnings.  Here is an invaluable shell command that allows you to find all packages that depend on a particular egg and version:
+
+    .. code-block:: bash
+
+        $ grep your.egg.name.here /home/vagrant/buildout-cache/eggs/*.egg/EGG-INFO/requires.txt
```

Comment:
I'm not wedded to an absolute path, if you can suggest a better way.  OTOH, this is not generic documentation, it is a training.